src/main/java/seedu/address/logic/parser/CliSyntax.java

```diff
@@ -9,6 +9,7 @@
     public static final Prefix PREFIX_DESCRIPTION = new Prefix("des/");
     public static final Prefix PREFIX_TAG = new Prefix("t/");
     public static final Prefix PREFIX_REMINDER = new Prefix("r/");
+    public static final Prefix PREFIX_RECURRING = new Prefix("rec/");
```

Comment:
Good way to not get confused with reminders. I'm wondering if r/ should be changed to rem/ for consistency. Might be too long but do consider it.